mobile/evidence/ios/Features/Tests/LoginTests/LoginEmailViewModelTests.swift

```diff
@@ -11,101 +11,102 @@ import XCTest
 
 final class LoginEmailViewModelTests: XCTestCase {
 
+    func testConfirmationPopupAppears() {
+        let viewModel = LoginEmailViewModel()
+        viewModel.confirmationPopupAppears()
+        let expectation = XCTestExpectation(description: "Confirmation popup should appear")
+        
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
+            XCTAssertEqual(viewModel.offSetY, 1000, "OffsetY should be updated to 1000")
+            expectation.fulfill()
+        }
+        wait(for: [expectation], timeout: 2)
+    }
+
     func testCloseButtonTapped() {
         let viewModel = LoginEmailViewModel()
         var closeButtonCalled = false
         viewModel.delegateCloseButtonTapped = {
             closeButtonCalled = true
         }
+        
         viewModel.closeButtonTapped()
         XCTAssertTrue(closeButtonCalled, "The delegate should be called")
     }
 
     func clearEmailInputTapped() {
         let viewModel = LoginEmailViewModel()
+        viewModel.emailInput = "email@valid.com"
+        viewModel.isLoginEmailButtonPressed = true
+
         viewModel.clearEmailInputTapped()
-<<<<<<< HEAD
-        XCTAssertTrue(viewModel.loginEmailInput.isEmpty, "Should be empty")
-=======
         XCTAssertTrue(viewModel.emailInput.isEmpty, "Should be empty")
->>>>>>> main
-        XCTAssertFalse(viewModel.isNextButtonPressed,  "Should set false")
```

Comment:
Great!

Reply:
When I merged the PR43 those conflicts disappeared. 🗡️